pycloak/middleware.py

```diff
@@ -199,14 +199,14 @@ def store_claim_on_user(self, request, user):
 
             # check for related fields on the user model
             rel_fields = attr["field"].split(".")
-            field = rel_fields.pop(0)
             obj = user
-            for rel_field in rel_fields:
-                obj = getattr(obj, rel_field)
+            field = None
```

Comment:
Obsolete?

Reply:
`field = None`? `field` is accessed later on. My IDE complains if I remove it, as `field` might be referenced before assignment. I guess it's a case of "explicit is better than implicit"?